content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -307,12 +307,14 @@ capabilities such as manipulating the network stack or accessing hardware device
 
 {{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
-In Windows, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod)
-by setting the `windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these
-pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used
-to perform administrative tasks as is done with Linux privileged containers. In order to use this feature, the
-`WindowsHostProcessContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled.
+This feature cannot be disabled with Kubernetes version 1.26 and later. In order to use this feature with Kubernetes 1.25 
+or earlier, the `WindowsHostProcessContainers` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) 
+must be enabled.
```

Comment:
```suggestion
```

The [timeless](https://developers.google.com/style/timeless-documentation) - and easy to maintain - approach here is to document as if the feature is stable and ready for use. The shortcode under the heading provides a big clue that older versions might not include the feature.

Reply:
I think we can just delete these 3 line - correct?
This feature will have been in beta for all currently supported versions of K8s (it went beta in 1.23). 
If folks want to see how they can disable it they can look at older doc versions.